packages/vite-plugin-cloudflare/src/runner/index.ts

```diff
@@ -161,6 +161,9 @@ export function createWorkerEntrypointWrapper(
 					server.accept();
 					createModuleRunner(this.env, server);
 
+					// TODO: here we need to add some validation, if createModuleRunner throws we want to return an error response
+					//       and process.exit or something from the other side
+
```

Comment:
This is something I noticed when working on this PR, but I don't want to scope creep my PR here, I'll create a separate issue for this validation (which is actually unrelated to this PR anyways)

Reply:
Issue created: https://github.com/flarelabs-net/vite-plugin-cloudflare/issues/59